core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -2059,7 +2059,9 @@ class SparkContext(config: SparkConf) extends Logging {
   /**
    * Shut down the SparkContext.
    */
-  def stop(): Unit = {
+  def stop(): Unit = stop(0)
+
+  def stop(exitCode: Int): Unit = {
```

Comment:
This is a public API and we need to document it properly.

Reply:
> This is a public API and we need to document it properly.

Updated